src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_BY_FIRST_LETTER = 'byLetter',
+  SORT_BY_GOOD_LENGTH = 'byLength',
+  RESET_SORTING = '',
+}
+
+function getSortedGoods(
+  goodsList: string[],
+  sortField: SortType,
+  reversed: boolean,
+) {
+  let goodsListCopy = [...goodsList];
```

Comment:
Prefer const over let and you don't need to add "list" in name is it's an array and everyone understand it